boards/arm/nrf52840_m2/nrf52840_m2.dts

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2022 Stavros Avramidis <stavros9899@gmail.com>
+ * Copyright (c) 2020 makerdiary
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "nrf52840_m2-pinctrl.dtsi"
+
+/ {
+	model = "Makerdiary nRF52840-M2 Module";
```

Comment:
This is not correct, also the name of the board should be changed, nrf52840-m2 is the name of the module, the name of the board/devkit is nrf52840 m2 developer kit